types/protobuf.go

```diff
@@ -13,9 +13,9 @@ type tm2pb struct{}
 func (tm2pb) Header(header *Header) *types.Header {
```

Comment:
we should figure out how to get rid of this file completely now. the types should be identical